index.html

```diff
@@ -14,11 +14,16 @@
         <a href="#" class="logo">My Logo</a>
         <nav>
             <button type="button" class="hamburger">&#9776;</button>
+            <ul class="nav-items">
+                <li><a href="">Portfolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
         </nav>
     </header>
-    <main>
+   
         <section class="content-section">
```

Comment:
Kindly note that your website has a horizontal bar at the bottom and the content starts to overflow out of the screen when you reduce the screen size a bit. This usually happens when you use a fixed size for your elements. I would highly recommend not using Fixed width and height where possible and making use of `flex` to make your website responsive to different screen sizes.
So kindly fix your website to use flex-box instead of fixed width and height, so that it's responsive for every screen size 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/188265660-2116af3d-bd45-4451-b713-140eee17e9f3.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188265681-49822fdf-f2f0-45e6-830c-084d84b80044.png)

Reply:
Good job following the advice of the previous reviewer